website/content/en/highlights/2021-08-24-vector-aggregator.md

```diff
@@ -0,0 +1,57 @@
+---
+date: "2021-08-24"
+title: "Moving the Aggregator chart to public beta"
+description: "The Vector Aggregator Helm chart is now publicly available"
+authors: ["spencergilbert"]
+pr_numbers: [8801]
+release: "0.16.0"
+hide_on_release_notes: false
+badges:
+  type: "announcement"
+  platforms: ["helm"]
+---
+
+We are happy to announce that our `vector-aggregator` chart is now publicly available. While
+we continue to iterate on and make improvements to the chart, we feel it is ready to get
+wider feedback from the community at large.
+
+We have also created a dedicated Discord channel, [#aggregator][discord], for support and questions.
+
+## Setup
```

Comment:
I could see linking to the quick start or installation guide you are working on in https://github.com/timberio/vector/pull/8489

Reply:
👍 I'm alright with replacing it here, it _will_ be duplicative once the installation instructions are there.